### DISCUSSION TITLE
https://github.com/appium/appium/blob/master/docs/packages/appium/docs throws 404

### DISCUSSION THREAD

**Discussion by durgapatro** [appium/appium#17260] [General]
Getting Page not found 404 error on opening the url - https://github.com/appium/appium/blob/master/docs/packages/appium/docs
Can someone please confirm if this page has been moved to some other location?

**Comment by jlipps** [comment]
Where did you find that URL? the correct one is https://github.com/appium/appium/tree/master/packages/appium/docs

**Reply by durgapatro** [reply]
Thank you Jonathan for your response. I was referring https://github.com/appium/java-client/blob/master/docs/v7-to-v8-migration-guide.md. and here links mentioned under Touch Actions section throws Page not found 404 error

https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/android-mobile-gestures.md
https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/ios/ios-xctest-mobile-gestures.md

Touch Actions
The TouchAction and MultiTouchAction classes have been deprecated. The support of these actions will be removed from future Appium versions. Please use [W3C Actions](https://w3c.github.io/webdriver/#actions) instead or the corresponding extension methods for the driver (if available). Check
https://www.youtube.com/watch?v=oAJ7jwMNFVU
https://appiumpro.com/editions/30-ios-specific-touch-action-methods
**https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/android-mobile-gestures.md**
**https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/ios/ios-xctest-mobile-gestures.md**
https://appiumpro.com/editions/29-automating-complex-gestures-with-the-w3c-actions-api for more details on how to properly apply W3C Actions to your automation context.

**Reply by jlipps** [reply]
replace `master` with `1.x` in those broken links and they will work. but yes, the idea is to use the w3c actions API moving forward.

**Comment by durgapatro** [comment]
Thank you:)
